org.eclipse.xtext.xtext.ui.tests/src/org/eclipse/xtext/xtext/ui/editor/hierarchy/XtextCallHierarchyBuilderTest.xtend

```diff
@@ -221,7 +222,7 @@ class XtextCallHierarchyBuilderTest extends AbstractHierarchyBuilderTest {
 					}
 					Model {
 						grammar: org.eclipse.xtext.ui.tests.editor.hierarchy.CallHierarchyBuilderTestLanguage
-						'name=ID' [295, 7]
+						'name=ID' [305, 7]
```

Comment:
The location of the `name` node changed its offset by 10 (length of @Override plus new line char)
![screenshot 28](https://cloud.githubusercontent.com/assets/265597/21645149/2f735228-d291-11e6-81a6-b74f383d2257.png)